libpng-1.2.56/build.sh

```diff
@@ -21,5 +21,7 @@ if [[ $FUZZING_ENGINE == "hooks" ]]; then
 fi
 set -x
 $CXX $CXXFLAGS -std=c++11 $SCRIPT_DIR/target.cc BUILD/.libs/libpng12.a $LIB_FUZZING_ENGINE -I BUILD/ -I BUILD -lz -o $EXECUTABLE_NAME_BASE
-$CXX $CXXFLAGS -std=c++11 $SCRIPT_DIR/target.cc BUILD/.libs/libpng12.a $LIB_FUZZING_ENGINE -I BUILD/ -I BUILD -lz -o $EXECUTABLE_NAME_BASE-structure-aware \
-  -include $SCRIPT_DIR/png_mutator.h -DPNG_MUTATOR_DEFINE_LIBFUZZER_CUSTOM_MUTATOR
+if [ -z $BUILD_STANDALONE_TARGET ]; then
```

Comment:
Hmm... So with this patch we effectively disable coverage builds for the custom mutator fuzzer.  Would it be better to fix the coverage build instead?

Reply:
Maybe! Let me look into it and I'll let you know